src/components/UpdateAppModal/BaseUpdateAppModal.js

```diff
@@ -33,31 +35,42 @@ class BaseUpdateAppModal extends PureComponent {
                     onSubmit={this.submitAndClose}
                     onClose={() => this.setState({isModalOpen: false})}
                     isVisible={this.state.isModalOpen}
+                    type={this.props.isSmallScreenWidth
+                        ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED : CONST.MODAL.MODAL_TYPE.CONFIRM}
```

Comment:
NAB typically we format multiline ternaries like this:

```javascript
const result = condition
    ? valueIfTrue
    : valueIfFalse;
```

Reply:
I'll update it.